apps/pie-docs/src/_11ty/shortcodes/tokensTable/handleTokenData.js

```diff
@@ -1,5 +1,3 @@
-const pieTokensMetadata = require('../../../tokensMetadata.json');
```

Comment:
These have been moved to `./utilities/tokens` so they can be shared between `tokenTable` and `simpleTable`